handler/src/main/java/io/netty/handler/ssl/OpenSslKeyMaterialManager.java

```diff
@@ -110,12 +110,12 @@ private void setKeyMaterial(long ssl, String alias) throws SSLException {
         }
     }
 
-    protected String chooseClientAlias(@SuppressWarnings("unused") OpenSslEngine engine,
+    protected String chooseClientAlias(@SuppressWarnings("unused") ReleasableOpenSslEngine engine,
                                        String[] keyTypes, X500Principal[] issuer) {
         return keyManager.chooseClientAlias(keyTypes, issuer, null);
     }
 
-    protected String chooseServerAlias(@SuppressWarnings("unused") OpenSslEngine engine, String type) {
+    protected String chooseServerAlias(@SuppressWarnings("unused") ReleasableOpenSslEngine engine, String type) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "engine". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)